Packs/ExportToXLSX/Scripts/ExportToXLSX/ExportToXLSX.py

```diff
@@ -0,0 +1,106 @@
+import traceback
+from typing import List, Optional
+
+import demistomock as demisto
+import xlsxwriter
+from CommonServerPython import *
+
+
+def write_data(sheet, data_item, data_headers, workbook, bold, border):
```

Comment:
add type annotations to all function arguments

Reply:
Done